crates/sui-framework/packages/sui-system/sources/validator_set.move

```diff
@@ -1091,7 +1091,7 @@ public(package) fun calculate_total_stakes(validators: &vector<Validator>): u64
     let mut i = 0;
     while (i < length) {
```

Comment:
Note: `validator_set` is not in the scope of this PR, this is expected a fix in the next one.